sdk/python/cmd/pulumi-language-python/testdata/setuppy/sdks/namespaced-16.0.0/setup.py

```diff
@@ -33,6 +33,7 @@ def readme():
       install_requires=[
           'parver>=0.2.1',
           'pulumi>=3.142.0,<4.0.0',
+          'pulumi_component>=13.3.7',
```

Comment:
This was missed in the initial PR that added PackageSpec.Dependencies. We handle it for `pyproject.toml` based SDKs, but not those using `setup.py`.

Reply:
Hmm feels like the conformance test should have caught this? If the program didn't reference the component provider directly would it have failed due to no direct library reference?

Reply:
Huh good point. Looking into this and I think the test passed because the *program* already depends on `pulumi_component`, so it gets installed in the venv, and the import inside the component code works at runtime.